packages/mui-material/src/Avatar/Avatar.test.js

```diff
@@ -20,6 +20,14 @@ describe('<Avatar />', () => {
     testDeepOverrides: { slotName: 'fallback', slotClassName: classes.fallback },
     testVariantProps: { variant: 'foo' },
     testStateOverrides: { prop: 'variant', value: 'rounded', styleKey: 'rounded' },
+    slots: {
+      root: {
+        expectedClassName: classes.root,
+      },
+      fallback: {
+        expectedClassName: classes.fallback,
+      },
+    },
```

Comment:
Cannot test the `img` slot together with `fallback` here because there is a logic within the component that dynamically render either one of them.